package.json

```diff
@@ -82,7 +83,7 @@
     "st": "^1.2.2",
     "watchify": "^3.11.1",
     "wd": "^1.11.2",
-    "webidl2js": "^9.2.2"
+    "webidl2js": "https://github.com/pmdartus/webidl2js.git#pmdartus/add-processor"
```

Comment:
**⚠️Use the released version of webidl2js before merging ⚠️** 
Using the WIP version to run the test suite on the CI.